core/templates/dev/head/pages/exploration_editor/simple_editor_tab/StatesToQuestionsService.js

```diff
@@ -174,6 +174,13 @@ oppia.factory('StatesToQuestionsService', [
       // current set of states to a list of questions.
       getQuestions: function() {
         return getQuestions();
+      },
+      getHumanReadableQuestionType: function(interactionId) {
+        for (key in SUPPORTED_INTERACTION_TYPES) {
```

Comment:
The syntax of forEach is SUPPORTED_INTERACTION_TYPES.forEach(...), not this. If you don't need to make use of the "i" then forEach is probably cleaner.